### DISCUSSION TITLE
Welcome to Dora Discussions!

### DISCUSSION THREAD

**Discussion by dinfuehr** [dinfuehr/dora#264] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by chaploud** [comment]
@dinfuehr 
Thank you for your great dora language!
This OSS project provides great insight into the creation of a new, fast programming language.
Is there a community (Discord, Slack, etc.) where you usually communicate about the dora language? I would like to join it.
Also, I would be very happy if you documented the points you were concerned with to speed up the language (arena memory allocation, etc.).

**Reply by chaploud** [reply]
Oh, there are maybe your blog posts about the dora language. I'll check it.
(https://dinfuehr.github.io/blog/dora-implementing-a-jit-compiler-with-rust/)
